oracle-server-ts/lib/index.ts

```diff
@@ -146,3 +148,25 @@ export function GetSignatures(eventName: string) {
     return <OracleResponse<OracleEvent>>response
   })
 }
+
+export function DeleteAnnouncement(eventName: string) {
+  console.debug('DeleteAnnouncement()')
+  validateString(eventName, 'DeleteAnnouncement()', 'eventName')
+
+  const m = getMessageBody(MessageType.deleteannouncement, [eventName])
+  return SendOracleMessage(m).then(response => {
+    // console.debug('DeleteAnnouncement response:', response)
+    return <OracleResponse<OracleEvent>>response
+  })
+}
+
+export function DeleteAttestation(eventName: string) {
+  console.debug('DeleteAttestation()')
```

Comment:
might be worth adding the event name of the attestation we are deleting to this log

Reply:
Done